kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating Oracle DB service and RCU Schema for a Fusion Middleware domain
+
+This sample demonstrates how to create an Oracle DB service on Kubernetes cluster and create RCU schema on the Oracle DB being used by a Fusion Middleware domain.  
+
+The directory contains the following sample scripts to start an Oracle DB service in default namespace, stop the Oracle DB service, create an RCU schema and drop the RCU schema
+
+```
+$ ./start-db-service.sh   
+Trying to pull repository container-registry.oracle.com/database/enterprise ... 
+12.2.0.1-slim: Pulling from container-registry.oracle.com/database/enterprise
+Digest: sha256:25b0ec7cc3987f86b1e754fc214e7f06761c57bc11910d4be87b0d42ee12d254
+Status: Image is up to date for container-registry.oracle.com/database/enterprise:12.2.0.1-slim
+deployment.extensions/oracle-db created
+service/oracle-db created
+[oracle-db-756f9b99fd-r6ghb] already initialized .. 
+Checking Pod READY column for State [1/1]
+NAME                         READY   STATUS    RESTARTS   AGE
+oracle-db-756f9b99fd-r6ghb   1/1     Running   0          3s
+NAME                         READY   STATUS    RESTARTS   AGE
+oracle-db-756f9b99fd-r6ghb   1/1     Running   0          4s
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP          16d
+oracle-db    NodePort    10.100.148.17   <none>        1521:30011/TCP   3s
+Oracle DB service is RUNNING with external NodePort [30011]
+
+```
+The parameters are as follows:
+
+```  
+  -p external NodePort for the Service (default is 30011) 
+```
+```
+The script create a Oracle DB Service in default Namesapce with default Credential that comes with the Oracle Database Slim image.
+$ kubectl get po
+NAME                         READY   STATUS    RESTARTS   AGE
+oracle-db-756f9b99fd-ch7xt   1/1     Running   0          33m
+$ kubectl get svc
+NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+oracle-db    NodePort    10.99.94.157   <none>        1521:30011/TCP   32m
+
+The DB Connection String is oracle-db.default.svc.cluster.local:1521/devpdb.k8s which can be used as  rcuDatabaseURL parameter to domain.input.yaml file while creating a Fusion Middleware domain in Operator Environment 
```

Comment:
is this text part of the output of something? if not, it should not be in the code block